package.json

```diff
@@ -61,7 +61,9 @@
 		"prern-bundle": "patch-package --patch-dir gutenberg/packages/react-native-editor/metro-patch",
 		"rn-bundle": "react-native bundle",
 		"postrn-bundle": "patch-package --reverse --patch-dir gutenberg/packages/react-native-editor/metro-patch",
-		"bundle": "npm run clean; npm run core prebundle && npm run bundle:js && npm run genstrings",
```

Comment:
I'd like to note that I removed `npm run core prebundle` from this command as now we're updating the i18n cache via the new `i18n:gutenberg` command.